bindings/python/test/coordinate/spherical/test_lla.py

```diff
@@ -10,146 +10,287 @@
+
 
-    lla: LLA = LLA.vector((5.0, 1.0, -1.0))
-
-    assert lla is not None
-    assert isinstance(lla, LLA)
-
-
-def test_coordinate_spherical_lla_comparators():
-    # Main Constructor
-    latitude: Angle = Angle(30.0, Angle.Unit.Degree)
-    longitude: Angle = Angle(30.0, Angle.Unit.Degree)
-    altitude: Length = Length(1000.0, Length.Unit.Meter)
-
-    # Construction
-    lla_1: LLA = LLA(latitude, longitude, altitude)
-    lla_2: LLA = LLA(latitude, longitude, Length(10.0, Length.Unit.Meter))
-
-    assert lla_1 == lla_1
-    assert lla_2 == lla_2
-    assert lla_1 != lla_2
-
-
-def test_coordinate_spherical_lla_is_defined():
-    # Main Constructor
-    latitude: Angle = Angle(30.0, Angle.Unit.Degree)
-    longitude: Angle = Angle(30.0, Angle.Unit.Degree)
-    altitude: Length = Length(1000.0, Length.Unit.Meter)
-
-    # Construction
-    lla: LLA = LLA(latitude, longitude, altitude)
-
-    assert lla.is_defined()
-    assert LLA.undefined().is_defined() is False
-
-
-def test_coordinate_spherical_lla_getters():
-    # Main Constructor
-    latitude: Angle = Angle(30.0, Angle.Unit.Degree)
-    longitude: Angle = Angle(30.0, Angle.Unit.Degree)
-    altitude: Length = Length(1000.0, Length.Unit.Meter)
-
-    # Construction
-    lla: LLA = LLA(latitude, longitude, altitude)
-
-    # get_latitude
-    lat = lla.get_latitude()
-
-    assert lat is not None
-    assert isinstance(lat, Angle)
-
-    # get_longitude
-    lon = lla.get_longitude()
-
-    assert lon is not None
-    assert isinstance(lon, Angle)
-
-    # get_altitude
-    alt = lla.get_altitude()
-
-    assert alt is not None
-    assert isinstance(alt, Length)
-
-
-def test_coordinate_spherical_lla_conversions():
-    # Main Constructor
-    latitude: Angle = Angle(30.0, Angle.Unit.Degree)
-    longitude: Angle = Angle(30.0, Angle.Unit.Degree)
-    altitude: Length = Length(1000.0, Length.Unit.Meter)
-
-    # Construction
-    lla: LLA = LLA(latitude, longitude, altitude)
-
-    # to_vector
-    vector = LLA.to_vector(lla)
-
-    assert vector is not None
-    assert isinstance(vector, np.ndarray)
-
-    # to_cartesian
-    cartesian = LLA.to_cartesian(lla, Length(3.0, Length.Unit.Meter), 3.0)
-
-    assert cartesian is not None
-    assert isinstance(cartesian, np.ndarray)
-
-    # to_string
-    string = lla.to_string()
-
-    assert string is not None
-    assert isinstance(string, String)
-    assert string == "[30.0 [deg], 30.0 [deg], 1000.0 [m]]"
+@pytest.fixture
+def lla_north_pole() -> LLA:
+    return LLA(
+        Angle(90.0, Angle.Unit.Degree),
+        Angle(0.0, Angle.Unit.Degree),
+        Length(1.0, Length.Unit.Meter),
+    )
+
+
+@pytest.fixture
+def lla_south_pole() -> LLA:
+    return LLA(
+        Angle(-90.0, Angle.Unit.Degree),
+        Angle(0.0, Angle.Unit.Degree),
+        Length(1.0, Length.Unit.Meter),
+    )
+
+
+class TestLLA:
+    def test_constructor_list_success(
+        self, latitude_float: float, longitude_float: float, altitude_float: float
```

Comment:
Add trailing `,` to make black pretty

```suggestion
        self,
        latitude_float: float,
        longitude_float: float,
        altitude_float: float,
```